.github/workflows/close-stale-issues.yml

```
@@ -0,0 +1,34 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
```

Same as previous comments and use `stale` as name to match workflow filename:

```suggestion
name: stale

on:
  schedule:
    - cron: '30 1 * * *'

permissions:
  issues: write
  pull-requests: write

jobs:
  stale:
    runs-on: ubuntu-latest
    steps:
      -
        name: Run
        uses: actions/stale@v9
        with:
```